packages/stripe_android/android/src/main/kotlin/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -243,7 +247,7 @@ class StripeSdkModule(reactContext: ReactApplicationContext, cardFieldManager: S
 
   @ReactMethod
   fun initPaymentSheet(params: ReadableMap, promise: Promise) {
-    val activity = currentActivity.activity
+    val activity = currentActivity.activity as FragmentActivity?
```

Comment:
@jonasbark This is the only relevant change. The latest master version of the react-native plugin uses a lot AppCompatActivity where It can be replaced by FragmentActivity (Notice the FlutterFragmentActivity does not inherit from AppCompatActivity so it will fail